examples/kinetic-electrons/wall+sheath-bc_boltzmann_loworder.toml

```diff
@@ -85,8 +85,8 @@ ascii_output = true
 [numerical_dissipation]
 #moment_dissipation_coefficient = 0.0001
 #moment_dissipation_coefficient = 1.0
-vpa_dissipation_coefficient = 0.002
-#vpa_dissipation_coefficient = 0.1
+#vpa_dissipation_coefficient = 0.002
+ion_vpa_dissipation_coefficient = 0.1
```

Comment:
You don't have to do anything for this comment, but to explain why in case you wonder about updating all the places `vpa_dissipation_coefficient` is used in example input files, etc.: that is the one other thing we would need to do for this change (i.e. update all the uses of `vpa_dissipation_coefficient` in the example input files and the tests). It's not useful to actually do that right now, since you're working on changing the structure like we discussed yesterday - once you've done that the change to the inputs will be to replace `[numerical_dissipation]` with `[ion_numerical_dissipation]` rather than `vpa_dissipation_coefficient` with `ion_vpa_dissipation_coefficient`, so it'd be a waste of time to change all the places where `vpa_dissipation_coefficient` appears in inputs, just to change them back again.

Reply:
I didn't see the notifications for these comments when you first sent them, sorry!! I hadn't realised that comments can happen here too...

Luckily, I have made sure not to do what you said to avoid, because of the future changes. I've committed your suggestions (separately, but now I see how to do the batch suggestions), and now I'm trying to run tests on my local updated code.